RPI/RpiDemo/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@
 # MOD_DEPS: (optional) module dependencies
 #
 ####
+add_compile_options(
+    -Wno-conversion
```

Comment:
If we don't fix these non-UT problems, should we capture them as issues?

Reply:
Captured in #3366.